csxtools/axis1/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .images import correct_images_axis
```

Comment:
please change the name to this module to `axis` from `axis1`.  specifically change `csxtools/axis1/` to  `csxtools/axis/`